src/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/synthetic.go

```diff
@@ -9,31 +9,32 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	envNodeType = "DT_NODE_SIZE"
-	envMaxHeap  = "DT_MAX_HEAP_MEMORY"
+	envNodeType   = "DT_NODE_SIZE"
+	envMaxHeap    = "DT_MAX_HEAP_MEMORY"
+	envLocationId = "DT_LOCATION_ID"
 
 	ChromiumCacheMountName = "chromium-cache"
 	chromiumCacheMountPath = "/var/tmp/dynatrace/synthetic/cache"
 
-	PersistentStorageMountName          = "persistent-storage"
-	synLogPersistentStorageMountPath    = "/var/log/dynatrace/synthetic"
-	synLogPersistentStorageMountSubPath = "syn-log"
-	synTmpPersistentStorageMountPath    = "/var/tmp/dynatrace/synthetic"
-	synTmpPersistentStorageMountSubPath = "syn-tmp"
+	ArchiveStorageMountName          = "archive-storage"
+	synLogArchiveStorageMountPath    = "/var/log/dynatrace/synthetic"
+	synLogArchiveStorageMountSubPath = "syn-log"
+	synTmpArchiveStorageMountPath    = "/var/tmp/dynatrace/synthetic"
+	synTmpArchiveStorageMountSubPath = "syn-tmp"
 
 	TmpStorageMountName             = "tmp-storage"
 	vucConfigTmpStorageMountPath    = "/var/lib/dynatrace/synthetic/config"
 	vucConfigTmpStorageMountSubPath = "vuc-config"
 	xvfbTmpStorageMountPath         = "/tmp"
 	xvfbTmpStorageMountSubPath      = "xvfb-tmp"
+	gatewayConfigMountSubPath       = "ag-config"
 )
 
 type SyntheticModifier struct {
-	dynakube dynatracev1beta1.DynaKube
+	dynatracev1beta1.DynaKube
```

Comment:
avoid "is a" connection, especially when using a dynakube (its messy, and we used to have it and was horrible) 
use "has a" connection (so what it was)

Reply:
Replacing the embedded `DynaKube` to identifier-specified.